tests/test_config_reader.py

```diff
@@ -0,0 +1,16 @@
+import yaml
+import os
+import pytest
+
+from pytentiostat.config_reader import parse_config_file, get_adv_params
+
+THIS_DIR = os.path.dirname(os.path.abspath(__file__))
+
+def test_parse_config_files():
+    confdir = os.path.join(THIS_DIR, 'static/')
+    config_data = parse_config_file(confdir)
+    assert isinstance(config_data, dict)
+    with pytest.raises(SystemExit):
+        config_data = parse_config_file("Not_Real_Directory")
+    with pytest.raises(SystemExit):
```

Comment:
nice.  good.

Did you also want to test that it is giving the right error message?  That could be an assert with the expected string.  It will look something like this:
```
    with pytest.raises(SystemExit) as pytest_exit_object:
            mymodule.will_exit_somewhere_down_the_stack()
    assert pytest_exit_object.type == SystemExit    # this gets the type of the error
    assert pytest_exit_object.value.code == 42       # this gets the exit code
    assert pytest_exit_object.<something> == "expected error message"     #not sure the command but I guess you can get the error message back somehow
```
```